crowd_anki/config/config_settings.py

```diff
@@ -49,17 +49,27 @@ class Properties(Enum):
         EXPORT_CREATE_DECK_SUBDIRECTORY = ConfigEntry("export_create_deck_subdirectory", True)
         IMPORT_NOTES_IGNORE_DECK_MOVEMENT = ConfigEntry("import_notes_ignore_deck_movement", False)
 
-    def __init__(self, addon_manager=None, init_values=None):
+    def __init__(self, addon_manager=None, init_values=None, window=None):
+        window = window or mw
```

Comment:
some nits here:   
* pass in the `profile_manager` instead of window, as we only need it
* order `addon_manager` and `profile_manager` to be adjacent in parameter list (as they are conceptually similar vs `init_values`)